src/Lender.sol

```diff
@@ -657,11 +659,18 @@ contract Lender {
         uint256 m,
         uint256 a,
         uint256 r,
+        Pendle.ApproxParams calldata g,
```

Comment:
Hmm, can we do a quick gas comparison here?

Guess as a param vs the static object. 

P.s. I'm assuming guess refers to a rough slippage limit in some way?

Reply:
Essentially id prefer to reduce the work for the frontend/library though really its just inputting a static object. If its a TINY amount of gas i dont mind hardcoding it in memory there